fmriprep/workflows/bold/confounds.py

```diff
@@ -266,7 +266,7 @@ def _pick_wm(files):
     return workflow
 
 
-def init_ica_aroma_wf(name='ica_aroma_wf', ignore_aroma_err=False):
+def init_ica_aroma_wf(name='ica_aroma_wf', ignore_aroma_err=False, aroma_melodic_dim=None):
```

Comment:
Please add in this docstring the new argument (they are listed under the `**Parameters**` section). Same comment stands for other workflow creation methods that have the argument. (unfortunately, workflow creation is not smart enough to infer arguments and/or their documentation).